### ISSUE TITLE
Don't produce a fatal error when attempting to activate this plugin before Woocommerce

### ISSUE THREAD

**Issue by cdhassell** [WPCV/wpcv-woo-civi-integration#58]
The behavior of this plugin is odd when handling the absence of the Woocommerce dependency. If I am testing a site and disable all plugins, and then re-enable them, this plugin (coming alphabetically before Woocommerce) hits a dependency error. This is okay, but it produces a fatal error and stops all further activations. So I  must manually re-activate all other plugins except this one, so it can be activated last. This seems unnecessarily difficult - it would be better for this plugin to simply skip its own activation with a warning flag and then allow the process for other plugins to continue. Thanks.

**Comment by christianwach** [comment]
@cdhassell I agree and have removed the heavy-handed dependency check with 2dbcb2b87ad6c38bd3ac575dd94e38b9343384dd in my development branch. It will be in v3.1.2 when that gets released.
